Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/pdf/ui/poster/CertificatePosterViewModel.kt

```diff
@@ -68,6 +68,21 @@ class CertificatePosterViewModel @AssistedInject constructor(
         }
     }
 
+    private fun deleteFile() = launch(context = dispatcher.IO) {
```

Comment:
I think we still need to use the `appScope` ? launch is triggered from viewModel scope and onCleared  there is a high chance that it get cancelled 😞

Reply:
launch(scope  = appScope, context = dispatcher.IO) look for `@AppScope` to see an example